packages/caver-transaction/src/transactionTypes/accountUpdate/accountUpdate.js

```diff
@@ -0,0 +1,141 @@
+/*
+    Copyright 2020 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const _ = require('lodash')
+const RLP = require('eth-lib/lib/rlp')
+const Bytes = require('eth-lib/lib/bytes')
+const AbstractTransaction = require('../abstractTransaction')
+const { TX_TYPE_STRING, TX_TYPE_TAG } = require('../../transactionHelper/transactionHelper')
+const Account = require('../../../../caver-account/src')
+const utils = require('../../../../caver-utils/src')
+
+function _decode(rlpEncoded) {
+    rlpEncoded = utils.addHexPrefix(rlpEncoded)
+    if (!rlpEncoded.startsWith(TX_TYPE_TAG.TxTypeAccountUpdate))
+        throw new Error(`Cannot decode to AccountUpdate. The prefix must be ${TX_TYPE_TAG.TxTypeAccountUpdate}: ${rlpEncoded}`)
+
+    const typeDettached = `0x${rlpEncoded.slice(4)}`
+    const [nonce, gasPrice, gas, from, rlpEncodedKey, signatures] = RLP.decode(typeDettached)
+    return {
+        nonce: utils.trimLeadingZero(nonce),
+        gasPrice: utils.trimLeadingZero(gasPrice),
+        gas: utils.trimLeadingZero(gas),
+        from,
+        rlpEncodedKey,
+        signatures,
+    }
+}
+
+/**
+ * Represents a account update transaction.
+ * Please refer https://docs.klaytn.com/klaytn/design/transactions/basic#txtypeaccountupdate to see more detail.
+ * @class
+ */
+class AccountUpdate extends AbstractTransaction {
+    /**
+     * decodes an RLP-encoded transation string and returns a AccountUpdate transaction instance.
+     *
+     * @param {string} rlpEncoded The RLP-encoded account update transaction.
+     * @return {AccountUpdate}
+     */
+    static decode(rlpEncoded) {
+        const decoded = _decode(rlpEncoded)
+        decoded.account = Account.createFromRLPEncoding(decoded.from, decoded.rlpEncodedKey)
+        return new AccountUpdate(decoded)
+    }
+
+    /**
+     * Creates a account update transaction.
+     * @constructor
+     * @param {object|string} createTxObj - The parameters to create a AccountUpdate transaction. This can be an object defining transaction information, or it can be an RLP-encoded string.
```

Comment:
```suggestion
     * @param {object|string} createTxObj - The parameters to create an AccountUpdate transaction. This can be an object defining transaction information, or it can be an RLP-encoded string.
```